tests/Php54/Php54Test.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Tests\Php54;
+
+use Symfony\Polyfill\Php54\Php54 as p;
+
+class Php54Test extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @runInSeparateProcess
+     */
+    public function testHttpResponseCodeDefaultValueIs200() {
+        $code = p::http_response_code(null);
```

Comment:
tests is this polyfill framework should be against the native function name (no `p::` prefix):
there is a phpunit test listener that will do the job of duplicating the test case, one testing the polyfill, the other testing the native implem if it's available.